backend/e4h-services/ingestion-service/app/utils/fieldplan_service_client.py

```
@@ -278,4 +298,69 @@ def unlink_fieldplan_facility(self, request_info: RequestInfo, fieldplan_id: str
+
+        files = {
+            "request": (None, json.dumps(bulk_request), "application/json"),
+            "excelFile": (file_name, file_bytes, "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"),
+        }
+
+        logger.trace(
+            f"Creating field plan template: fieldPlanId={field_plan_id}, systemType={system_type}, "
+            f"totalCapacity={total_capacity}"
+        )
+        try:
+            response = requests.post(url, files=files)
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)